src/navbar/theme.scss

```diff
@@ -4,10 +4,8 @@
   z-index: 2;
   box-sizing: border-box;
   width: 100%;
-  height: 81px;
+  height: fit-content;
```

Comment:
We can provide a fallback to some fixed height since fit-content not supported in EDGE and IE